packages/app-bridge/src/AppBridgeBlock.ts

```diff
@@ -91,4 +101,9 @@ export interface AppBridgeBlock extends AppBridgeBase {
     getBulkDownloadBySignature(signature: string): Promise<BulkDownload>;
 
     getPrivacySettings(): PrivacySettings;
+
+    dispatch<Command extends keyof BlockCommandResponse>(
+        command: Command,
+        payload?: BlockCommandPayload[Command],
+    ): BlockCommandResponse[Command];
```

Comment:
We should definitely discuss this. I am checking how this would look as a change with the existing methods planned to be moved into Commands.
For now, though, can we go ahead with the current syntax and if we plan to go ahead with this, we can do it as a separate iteration?

Reply:
Did you mean something like this with the above suggestion?
```
// Helper method:

const openAssetChooser = (options?: OpenAssetChooserOptions) => {
    return {
        eventName: 'AssetChooser.Open',
        payload: {
            options: options
        }
    }
}

// Now the helper methods get passed into the `dispatch` for readability
const assetChooser = appBridge.dispatch(openAssetChooser);

assetChooser.close(); // assuming this stays the same?
```

We can discuss further in our call later.

Reply:
> Did you mean something like this with the above suggestion?

`appBridge.dispatch(openAssetChooser())`*
but yes 

> For now, though, can we go ahead with the current syntax and if we plan to go ahead with this, we can do it as a separate iteration?

We are talking about half an hour to an hour of work to change it, so let's do it now ;)

Reply:
Updated as per our discussion yday